pkg/steps/multi_stage_test.go

```diff
@@ -547,6 +548,9 @@ func TestRun(t *testing.T) {
 }
 
 func TestArtifacts(t *testing.T) {
+	timeSecond = time.Nanosecond
+	defer func() { timeSecond = time.Second }()
```

Comment:
Shouldn't this result in an error?

Reply:
It does but we only log it: https://github.com/openshift/ci-tools/pull/1030/files#diff-892295443de5592354f856e247809216R452

Reply:
github is so great, doesn't expand the are the link is in, this one should work: https://github.com/openshift/ci-tools/blob/77d4e0ac850628341ffc1075527bee60a4d9dbc5/pkg/steps/artifacts.go#L452